src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -1102,11 +1186,22 @@ public async Task<IActionResult> ConfirmSubmitApplication(SubmitApplicationViewM
             var organisationDetails = await _apiClient.GetOrganisationByUserId(User.GetUserId());
             var providerRoute = await _qnaApiClient.GetAnswerByTag(model.ApplicationId, RoatpWorkflowQuestionTags.ProviderRoute);
 
+            var application = await _apiClient.GetApplication(model.ApplicationId);
+            foreach(var sequence in application.ApplyData.Sequences)
+            {
+                var applicationSequence = await _qnaApiClient.GetSequenceBySequenceNo(model.ApplicationId, sequence.SequenceNo);
+                var sections = await _qnaApiClient.GetSections(model.ApplicationId, applicationSequence.Id);
+                
+                applicationSequence.Sections = sections.ToList();
+                sequence.NotRequired = SequenceNotRequired(applicationSequence, _notRequiredOverrides);
+            }
+
             var submitApplicationRequest = new Application.Apply.Submit.SubmitApplicationRequest
             {
                 ApplicationId = model.ApplicationId,
                 ProviderRoute = int.Parse(providerRoute.Value),
-                SubmittingContactId = User.GetUserId()
+                SubmittingContactId = User.GetUserId(),
+                ApplyData = application.ApplyData
```

Comment:
See previous comment about committing ApplyData here. I would have expected this in Create Application